src/components/Header.js

```diff
@@ -41,6 +43,14 @@ export default function(store){
         }
       }
     },
+    mounted() {
```

Comment:
ok yup! debounce is good. I've traditionally done something like that, but saw a vue implementation that used `setInterval` and thought it sorta made sense since sticky elements aren't expected to be super responsive.

But yeah you are probably right in retrospect that was maybe a ¯\_(ツ)_/¯ idea lol